index.rst

```diff
@@ -0,0 +1,8 @@
+**Registrazione sito primario PA**
+
+.. toctree::
+  :maxdepth: 3
+  :caption: Indice dei contenuti
+
+  sommario.rst
```

Comment:
Una elenco dei contenuti è già generato dalla direttiva `toctree`. Un breve sommario (nel senso di summary - riassunto) è opportuno all'inizio di ogni capitolo. Vedi: https://docs.italia.it/italia/docs-italia/docs-italia-guide/it/bozza/appendice-2/riassunto.html